bga_database/static/js/chart_helper.js

```diff
@@ -142,7 +142,22 @@ var ChartHelper = {
       chart: {
         type: 'bar'
       },
-      colors: ['#004c76', '#c84747', '#fd0', '#67488b', '#1a9b5b', '#343a40'],
+      tooltip: {
+        formatter: function() {
+          return this.series.name;
+        }
+      },
+      legend: {
+        floating: true,
+        verticalAlign: 'top',
+        y: 30
+      },
+      colors: ['#0B2F42', '#004C76', '#245D8C', '#538BC2', '#82BEED', '#758892'],
```

Comment:
I really like the revision to colors! The second and third are a little close... what do you think about a darker blue, like `#023f62`, for the second?